### PR TITLE
Standardize pathogen title names 

### LINKED ISSUE #917: Standardize casing of pathogen titles
_follow-up to https://github.com/nextstrain/nextstrain.org/pull/901#discussion_r1638611959_

## Context

Pathogen titles are shown in two places:

### 1. Splash page

image

### 2. Core pathogens page

image

## Description

The titles are a mix of title case and lower case both in the context of each page (e.g. Chikungunya vs. seasonal influenza) and between the two pages (e.g. measles vs. Measles). It should be one or the other for consistency. Which one?

### PR DESCRIPTION
**Pull request by victorlin**
[_preview_](https://nextstrain-s-victorlin--abdljp.herokuapp.com)

## Description of proposed changes

Keeping things consistent.

## Related issue(s)

- Fixes #917

## Checklist

- [x] Checks pass
- [x] ~Check if changes affect the [resource index JSON revision](https://docs.nextstrain.org/projects/nextstrain-dot-org/en/latest/resource-collection.html#resource-index-revisions)~

### CODE REVIEW

File: static-site/content/resource-listing.yaml [hunk 1]
```diff
@@ -62,23 +62,23 @@ coreShowcase:
     img: ncov.png
     filters:
       - ncov
-  - name: Seasonal Flu
+  - name: Seasonal influenza
     img: seasonalinfluenza.png
     filters:
       - seasonal-flu
   - name: Avian Flu
```
Comment (joverlee521) [inline static-site/content/resource-listing.yaml:69]:
```suggestion
  - name: Avian Flu
  - name: Avian Influenza
```

File: static-site/content/resource-listing.yaml [hunk 2]
```diff
@@ -58,12 +58,39 @@ coreGroupDisplayNames:
...
+      - mpox
+  - name: RSV
+    img: rsv1.png
+    filters:
+      - rsv
+  - name: Dengue
+    img: dengue.png
+    filters:
+      - dengue
+  - name: Ebola (West Africa)
```
Comment (trvrb) [inline static-site/content/resource-listing.yaml:85]: Switch to `Ebola in West Africa`

I don't like the unnecessary parentheticals.

**Comment by trvrb** [comment]
Thanks for fixing this up @victorlin. Looks good to me with the swaps from "flu" to "influenza" and from "(West Africa)" to "in West Africa".

File: static-site/content/resource-listing.yaml (hunk 1 above)
Comment (victorlin) [reply static-site/content/resource-listing.yaml]: Fixed in [force-push](https://github.com/nextstrain/nextstrain.org/compare/36af2e91337f13b1a1da138d2dfef5d040b37288..833b1146cfdbb9af1e8eb207a1eb964b76967ae4)

File: static-site/content/resource-listing.yaml (hunk 2 above)
Comment (victorlin) [reply static-site/content/resource-listing.yaml]: Fixed in [force-push](https://github.com/nextstrain/nextstrain.org/compare/36af2e91337f13b1a1da138d2dfef5d040b37288..833b1146cfdbb9af1e8eb207a1eb964b76967ae4)

**Pull request merged**